index.ts

```diff
@@ -241,11 +237,15 @@ try {
         .join(',\n') +
       '\n```';
 
+    const defaultMessage = globalInstallMessage
+      ? `Test the snapshot by intalling your package globally:`
+      : `Test the snapshot${multiple ? 's' : ''} by updating your \`package.json\` with the newly published version${multiple ? 's' : ''}:`;
```

Comment:
@isaacroldan I think this is a better developer experience for global packages. I don't think you need to use `customMessagePrefix` for your `snapit` implementation.